lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -64,6 +102,16 @@ export default {
 				return Object.values(BADGE_COLORS).includes(value);
 			},
 		},
+		icon: {
+			type: String,
+			required: true,
+			validate: (icon) => Object.values(ICONS).includes(toRaw(icon)),
+		},
+		size: {
```

Comment:
it isn't `size` of the component, so name is misleading.

you force one `size` for icon in CSS (which isn't optimal), so why do you expose this prop?

Reply:
I've removed this prop, CSS styles and I've moved icon size directly into component, passing certain value as a string, like this `<ds-icon v-if="icon" class="badge__icon" :icon="icon" size="xx-small" />`. Is that allright?

Reply:
I commented it inline. You should use const for `size` in the template